lib/dal/src/component.rs

```diff
@@ -4054,7 +4054,10 @@ impl Component {
         let mut sockets = Vec::new();
         for mut comp_socket in schema_sockets.clone() {
             if let Some(is_managed) = comp_socket.is_management {
```

Comment:
As a future note you can do:

`if let Some(true) = comp_socket.is_management` here and avoid the second `if` inside